src/store/store.js

```diff
@@ -0,0 +1,11 @@
+import thunk from 'redux-thunk';
+import { createStore, applyMiddleware, compose } from 'redux';
+import reducer from './rootReducer';
+
+const enhancers = compose(
+  applyMiddleware(thunk),
+  window.devToolsExtension && window.devToolsExtension(),
```

Comment:
+  `window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()`

Reply:
Duly noted @nadralia I will be using `edux-devtools-extension`. Thanks.